### ISSUE TITLE
Isolate text from a single column with bash

### ISSUE THREAD

**Issue by yaaminiv** [RobertsLab/resources#595]
I have a file with output from intersecting the *C. virginica* mRNA GFF with DMRs, [found here](
https://github.com/fish546-2018/yaamini-virginica/blob/master/analyses/2018-11-01-DML-and-DMR-Analysis/2018-11-07-DMR-mRNA.txt). In the last column, there is information about the gene the DMR was found in. Specifically, there is a "product=" designation that describes the protein generated from this mRNA coding region. For example:

```
ID=rna48;Parent=gene35;Dbxref=GeneID:111114201,Genbank:XM_022452489.1;Name=XM_022452489.1;Note=The sequence of the model RefSeq transcript was modified relative to this genomic sequence to represent the inferred CDS: inserted 2 bases in 2 codons;exception=unclassified transcription discrepancy;gbkey=mRNA;gene=LOC111114201;model_evidence=Supporting evidence includes similarity to: 4 Proteins%2C and 99%25 coverage of the annotated genomic feature by RNAseq alignments%2C including 9 samples with support for all annotated introns;product=vacuolar protein sorting-associated protein 13B-like;transcript_id=XM_022452489.1
```

For each line, I want to isolate the product information in this column so I can easily describe/visualize the different product functions. I tried just unfolding this column in Excel using semicolon delimiters, but each line has a different amount of information, so the product information doesn't end up in the same column throughout the document. Is there any way to extract the product information and paste it into a new column? Could I specify `product=` as a delimiter to separate the information into a new column?

**Comment by kubu4** [comment]
This will work on what you posted above (enter in Terminal):

```awk -F';' '{print $10}' <input_file>```

The command uses a program called ```awk``` and sets the delimiter as a semi-colon. It then prints filed number 10 from the input file provided.

However, when you say this, what exactly do you mean:

> but each line has a different amount of information

Does this mean there is a different number of semi-colons used to delimit the information? If yes, then the above code won't work for all of your cases.

**Comment by yaaminiv** [comment]
> Does this mean there is a different number of semi-colons used to delimit the information?

Yes, there are a different number of semicolons used to delimit information in each line (artifact of the GFF)

**Comment by yaaminiv** [comment]
Is it possible to create a custom delimiter? Or only extract everything after a certain phrase in a line?

**Comment by yaaminiv** [comment]
I took the easy way out and used Excel:

I created a duplicate column, replaced `product=` with `>`, then used `>` as a delimiter to separate product information from the rest of the column. I then used `;` as a delimiter with the column I just generated to separate the product information from the transcript ID. Finally, I deleted columns without product information that I generated throughout this process. My final document looks like this:

![screen shot 2019-02-25 at 4 14 47 pm](https://user-images.githubusercontent.com/22335838/53377774-022cce80-3918-11e9-8d64-591bf78c068d.png)
